src/phoenix/config.py

```diff
@@ -331,5 +338,20 @@ def get_web_base_url() -> str:
     return get_base_url()
 
 
+class LoggingMode(Enum):
+    DEFAULT = "default"
+    STRUCTURED = "structured"
```

Comment:
can we just use structured logs all the time and have a serializer? That's way simpler

Reply:
Synced offline